tests/Functional/Entity/FTransformerTest.php

```diff
@@ -0,0 +1,479 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Tests\Functional\Entity;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Id\AbstractIdGenerator;
+use Doctrine\ORM\Id\AssignedGenerator;
+use Doctrine\ORM\Mapping\ClassMetadataInfo;
+use Sonata\PageBundle\Model\TransformerInterface;
+use Sonata\PageBundle\Tests\App\AppKernel;
+use Sonata\PageBundle\Tests\App\Entity\SonataPageBlock;
+use Sonata\PageBundle\Tests\App\Entity\SonataPagePage;
+use Sonata\PageBundle\Tests\App\Entity\SonataPageSite;
+use Sonata\PageBundle\Tests\App\Entity\SonataPageSnapshot;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+/**
+ * @phpstan-import-type PageContent from TransformerInterface
+ * @phpstan-import-type BlockContent from TransformerInterface
+ */
+final class FTransformerTest extends KernelTestCase
```

Comment:
Typo 
```suggestion
final class TransformerTest extends KernelTestCase
```
and in the file name

Reply:
i named the class to not be confused with the other TransformerTest (especially in the IDE)

Reply:
It's not needed, that's the purpose of different namespace. We don't use such prefix in sonata code base.